content/posts/blog/first-paris-xmpp-meetup.md

```diff
@@ -0,0 +1,13 @@
+---
+title: First Paris XMPP/Jabber meetup, March 28th
+date: 2017-03-23 23:59
+author: Nÿco
+blog_id: blog
+category: Summits, Conferences, And Meetups
+---
+
+After Berlin, Stockholm, London, here is Paris. The trend of new XMPP meetups in major cities in Europe is confirmed.
```

Comment:
"and London", maybe? I think it reads better, but I'm not srue if it's actually necessary.